lib/src/widgets/sliver.dart

```diff
@@ -182,7 +192,7 @@ class SliverVariableSizeBoxAdaptorElement extends RenderObjectElement
         as SliverVariableSizeBoxAdaptorParentData?;
 
     // set keepAlive to true in order to populate the cache
-    if (newParentData != null) {
+    if (addAutomaticKeepAlives && newParentData != null) {
```

Comment:
I think there is a problem here.

I used your fork on for a `SliverStaggeredGrid.countBuilder`, I gave `addAutomaticKeepAlives: true` as an parameter.

But it change nothing.

When debbugging, the `addAutomaticKeepAlives` on the line above never went `true`.

Maybe there is a problem when passing props to children

Reply:
`keepAlives: true` is not a parameter you need to pass to the grid constructor, that is for testing it in the script I have provided above.

The parameter on the constructor is `addAutomaticKeepAlives: true`,
but there is no point in setting it to `true` since it is already `true` by default, so I am not sure why you are expecting a change there.

If you want to _disable_ keepAlives, then use `addAutomaticKeepAlives: false`.

I have also just tested this and it works fine when debugging.
Putting a breakpoint on line `196` shows that it gets called.

With keepAlives / cache:
```dart
 StaggeredGridView.countBuilder(),
```

Without keepAlives / cache
```dart
StaggeredGridView.countBuilder(
            addAutomaticKeepAlives: false,
          ),
```